pkg/alertmanager/types.go

```diff
@@ -400,4 +401,12 @@ type victorOpsConfig struct {
 	CustomFields      map[string]string `yaml:"custom_fields,omitempty" json:"custom_fields,omitempty"`
 }
 
+type msTeamsConfig struct {
+	SendResolved *bool             `yaml:"send_resolved,omitempty"`
+	WebhookURL   string            `yaml:"webhook_url"`
+	Title        *string           `yaml:"title,omitempty"`
+	Text         *string           `yaml:"text,omitempty"`
```

Comment:
better to stick with the Alertmanager struct and use values instead of pointers.
```suggestion
	Title        string           `yaml:"title,omitempty"`
	Text         string           `yaml:"text,omitempty"`
```